aten/src/ATen/native/TensorShape.cpp

```diff
@@ -627,23 +627,23 @@ static void check_t(const Tensor& self, const char *fn) {
   if (self.is_sparse()) {
     int64_t sparse_dim = self.sparse_dim();
     int64_t dense_dim = self.dense_dim();
-    AT_CHECK((sparse_dim == 2 || sparse_dim == 1) && dense_dim == 0,
-             fn, " expects a tensor with 1 or 2 sparse and 0 dense dimensions, but got ",
+    AT_CHECK(sparse_dim <= 2 && dense_dim == 0,
+             fn, " expects a tensor with 0 or 1 or 2 sparse and 0 dense dimensions, but got ",
```

Comment:
I would change "0 or 1 or 2" to "<= 2 throughout this PR.

Reply:
done